README.md

```diff
@@ -0,0 +1,89 @@
+[![Python Requirements Status](https://requires.io/github/etalab/udata/requirements.png?branch=master)](https://requires.io/github/etalab/udata/requirements/?branch=master)
+[![JavaScript Dependencies Status](https://david-dm.org/etalab/udata.svg)](https://david-dm.org/etalab/udata)
+[![Join the chat at https://gitter.im/etalab/udata](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/etalab/udata?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+uData is an open and social data hub. This currently a work in progress and API is subject to change until the first official release.
+
+
+Compatibility
+-------------
+
+udata requires Python 2.7, MongoDB 3.2+, ElasticSearch 1.7 and Redis.
+
+
+Installation
+------------
+
+### System dependencies
+
+sudo apt-get install -y --no-install-recommends tar git wget curl build-essential pkg-config  python python-dev python-pip libjpeg-dev zlib1g-dev libpng12-dev libtiff5-dev libfreetype6-dev liblcms2-dev libopenjpeg-dev libwebp-dev libpng12-dev libxml2-dev libxslt1-dev liblzma-dev libyaml-dev && sudo pip install virtualenv cython
+
+> MacOSX: checkout Homebrew for installing dependencies.
+
+
+### Installing the project
+
+Let's call the root project `$WORKSPACE`.
+
+    mkdir $WORKSPACE && cd $_
+
+Retrieve the sources
+
+    git clone https://github.com/etalab/udata.git
+Install your [virtualenv](https://virtualenv.readthedocs.org/en/latest/)
+
+    virtualenv --python=python2.7 venv
+    source venv/bin/activate
+    pip install -r udata/requirements/develop.pip
+
+
+### Installing 3rd party services
+
+The project depends on ElasticSearch, Redis and MongoDB. We will use [docker-compose](https://docs.docker.com/compose/) to manage all that.
+[Install Docker-Compose for your system](https://docs.docker.com/compose/install/). On linux you should also [create a _docker_ group](https://docs.docker.com/engine/installation/linux/ubuntulinux/#create-a-docker-group). On Windows and MacOSX you will have to using a [docker-machine](https://docs.docker.com/machine/overview/).
+
+Then start the services:
+
+    cd data
+    docker-compose up
+
+On the very first run it will download and install images which takes a while.
+
+> Test your _docker-compose_ is running successfully entering `curl http://locahost:9200` It should output a JSON search response. If you have no output at all for too long, check the [IPv6 possible issue](https://github.com/docker/docker/issues/2174#issuecomment-35697655).
+
+
+#### Installing natively
+
+If you really don't want to use docker you can install MongoDB 3.2, ElasticSearch 1.7 with analysis-icu plugin, and Redis 3.0 on your way.
+
+Now open your browser to http://localhost:7000/. It's empty and ugly, yet it works!
+
+
+### Building CSS
+
+To build the CSS run in a dedicated terminal:
+
+    npm assets:watch
```

Comment:
mistake, it's `npm run assets:watch`.
`inv watch` calls `npm run assets:watch
`